addon/serializers/socket.js

```diff
@@ -17,8 +16,23 @@ const {
 } = DS;
 
 export default RESTSerializer.extend({
-  extractFindQuery(store, type, payload) {
-    return this.extractArray(store, type, payload.payload);
+  isNewSerializerAPI: true,
+
+  payloadWithMeta(payload) {
+    let meta;
+    let customPayload = payload.payload;
+
+    if (payload && payload['meta'] !== undefined) {
```

Comment:
@marpo60 you are checking if `payload` exists here but if it wont exist line 23 and 28 will also fail, so if there are cases when payload is not defined i suggeset to make this area of code safer